botfront/imports/ui/components/settings/fileImporters.js

```diff
@@ -31,7 +31,7 @@ export const handleImportDomain = (files, { projectId, fileReader: [_, setFileLi
     if (!files.length) return;
     setImportingState(true);
     files.forEach(({
-        slots, templates, filename, lastModified,
+        slots, responses, filename, lastModified,
```

Comment:
Are imports from old rasa versions that are using template instead of responses are still supported ?

Reply:
Yes, cf. loadDomain.js, where legacy "templates" and modern "responses" are output to this here attribute.